### PR TITLE
Define how acronyms are rendered in camel case

### PR DESCRIPTION
**Pull request by john-n-smith**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 2. Short array syntax (`[]` instead of `array()`) MUST be used.
 3. All PHP code MUST adhere to the [PSR-4 Autoloading](http://www.php-fig.org/psr/psr-4/) Standard.
 4. Variable names MUST be camelCase and not snake_case or StUdLyCaSe or UpperCamelCase.
+  1. Acronyms MUST be treated as normal words. `xmlHttpRequest` not `XMLHTTPRequest`.
```
Comment (biggianteye) [inline README.md]: Is it worth differentiating between class (`XmlHttpRequest`) and variable (`xmlHttpRequest`)?

Comment (wpillar) [inline README.md]: @john-n-smith I think @biggianteye is right, we should mention what to do with acronyms in classes too.

Comment (john-n-smith) [inline README.md]: We don't seem to define class naming behaviour. Is this inherited from PSR-2?

Comment (wpillar) [inline README.md]: PSR-1 actually: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md#1-overview

Neither PSR-1 or PSR-2 specific a standard for variable naming, which is why it's explicitly defined in ours.

**Comment by john-n-smith** [comment]
@wpillar, @biggianteye. Added a definition for class names with a note about acronyms. I've used `UpperCamelCase` over the PSR's `StudlyCase`, which according to wikipedia is arbitrary (and we use 'UpperCamelCase' elsewhere).

**Pull request merged**